### PR TITLE
Re-check asset still exists after prompts (in case revoked)

### PR DESCRIPTION
**Pull request by gabriel**
@cjb

Minor tweak to re-check the asset still exists after all prompts.
This allows an update to be revoked immediately.

I think a small tweak for a potentially very helpful feature?

### CODE REVIEW

**Comment by cjb** [comment]
@keybase/react-hackers 

This is fine with me, but I remember someone pointed out that not being able to apply an update that you downloaded earlier after you go offline will be counter-intuitive.

Still, :thumbsup: from me, perhaps wait for a golang review from e.g. @patrickxb.

File: go/updater/updater.go [hunk 1]
```diff
@@ -335,6 +335,26 @@ func (u *Updater) unpack(filename string) (string, error) {
...
+	if err != nil {
+		return err
+	}
+	if resp == nil {
+		return fmt.Errorf("No response")
+	}
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("Update is no longer available (%d)", resp.StatusCode)
+	}
+	defer libkb.DiscardAndCloseBody(resp)
```
Comment (patrickxb) [inline go/updater/updater.go]: Do you need defer on this?  And does `http.Head` get a body, or is this just to be safe?

**Comment by patrickxb** [comment]
LGTM, one small question.

File: go/updater/updater.go (hunk 1 above)
Comment (gabriel) [inline go/updater/updater.go]: I think just to be safe... I'll move the defer before the status code check (in which case it does need to be a differ).. thx!

**Pull request merged**